commands/build-char-refs-command.js

```diff
@@ -1,79 +1,23 @@
-var http = require('http');
-var fs = require('fs');
-
-var outFile = require('path').resolve(__dirname, '../src/html5-named-char-refs.ts');
-
-// lifted from jshint
-var UNSAFE = /[\u0000-\u001f\u007f-\u009f\u00ad\u0600-\u0604\u070f\u17b4\u17b5\u200c-\u200f\u2028-\u202f\u2060-\u206f\ufeff\ufff0-\uffff]/;
+"use strict";
+const entities = require("./codegen/entities");
+const codegen = require("./codegen");
 
 module.exports = {
-  name: 'build-char-refs',
-  works: 'insideProject',
-  description: 'Rebuild "html5-named-char-refs" from "http://www.w3.org/TR/html5/entities.json"',
+  name: "build-char-refs",
```

Comment:
Seems like prettier is configured to use single quotes, but this was still changed?